packaging/test_irods_client_rest_cpp.py

```diff
@@ -350,6 +350,34 @@ def test_list_with_accoutrements(self):
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
 
+    def test_query_handles_case_insensitivity__issue124(self):
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                case_sensitive = 0
```

Comment:
codacy says this is not used.    which is ... concerning?   seems we should use it in 367/368?